README.md

```diff
@@ -1 +1,49 @@
-# Cyclone
\ No newline at end of file
+<h1 align="center">
+	<br>
+	<img width="400" src="docs/images/logo.jpeg" alt="cyclone">
+	<br>
+	<br>
+</h1>
+
+Cyclone is a cloud native workflow engine run in Kubernetes. It's implemented with Kubernetes CRD (Custom Resource Definition), with no extra database components required. Once deployed, you will get one cloud-scale workflow engine ready for your workflows.
+
+Cyclone defines each workflow stage as a pod, and assemble them to workflow by capturing dependencies between stages as DAG. 
+
+## Features
+- **Kubernetes Native**: Cyclone run on any Kubernetes clusters without any other dependencies.
+- **CI/CD**:  Cyclone supports CI/CD pipeline natively by templates.
+- **AI Workflow**: ...
+- **DAG**: Cyclone supports any DAG workflow.
+- **Resources**: Abstract input/output of workflow as resource and support any resource types as long as resource resolver image provided. For example, SCM, Docker Registry, S3.
```

Comment:
Resource is the implementation, maybe can not be understood by readers.
What about:

**Pluggability**: Cyclone can easily integrate with external systems, such as SCM, Docker Registry, S3, etc.